src/OrbitGl/CaptureWindow.cpp

```diff
@@ -392,16 +395,16 @@ void CaptureWindow::Draw() {
   }
 
   if (time_graph_ != nullptr) {
-    // We need to update the visible track list to know the total height of them (used in the
-    // viewport).
-    time_graph_->GetTrackManager()->UpdateTracksForRendering();
-
-    viewport_.SetWorldExtents(viewport_.GetScreenWidth(), time_graph_->GetHeight());
-
+    time_graph_->UpdateLayout();
     UpdateChildrenPosAndSize();
+    // HACK: This needs to be done as long as TimeGraph does not properly report its children.
```

Comment:
FWIW I'm working on getting rid of this as well. I consider this still part of the current bug I'm working on, but in case I need to put this work on hold I'll create a separate bug.